vllm/engine/metrics.py

```diff
@@ -165,6 +216,26 @@ def _log_prometheus(self, stats: Stats) -> None:
         self.metrics.counter_generation_tokens.labels(**self.labels).inc(
             stats.num_generation_tokens)
 
+        # Add to request counters.
+        for finished_reason, count in stats.finished_reason_counter.items():
+            self.metrics.counter_request_success.labels({
+                **self.labels,
+                "finished_reason":
+                finished_reason,
```

Comment:
Does this not raise an error because `finished_reason` was not specified when this metric was constructed?